schema/methods_directory/physical/psp/file_data_item.json

```diff
@@ -54,7 +54,10 @@
         },
         "apps": {
             "type": "array",
-            "description": "simulation engines that can use this pseudopotential"
+            "description": "simulation engines that can use this pseudopotential",
```

Comment:
An Application is an object, we should clarify that this field is for the application names